### ISSUE TITLE
Master Child Navigation - Update?

### ISSUE THREAD

**Issue by thevirus20** [madcyph3r/AdvancedMaterialDrawer#62]
Hi,
I found that you have updated the code, but what about Master Child Navigation?
Have you implemented this type of navigation in your code? See #26 

Thanks.

**Comment by madcyph3r** [comment]
No, i haven't insert the code yet. Can you tell me, what you can do with the master child navigation, you can't do now.
